testdata/default/cheats/CloneAccount.t.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.18;
+
+import "ds-test/test.sol";
+import "cheats/Vm.sol";
+
+contract Source {
+    uint256 public a;
+    address public b;
+    uint256[3] public c;
+    bool public d;
+
+    constructor() {
+        a = 100;
+        b = address(111);
+        c[0] = 222;
+        c[1] = 333;
+        c[2] = 444;
+        d = true;
+    }
+
+    function getC(uint256 index) public returns (uint256 val) {
+        val = c[index];
+    }
```

Comment:
this is identical to auto-generated `c` getter

Reply:
removed in https://github.com/foundry-rs/foundry/pull/9048/commits/77e0033c5aab132819b24b179cadc8dc6d4eb865